sdk/redisenterprise/Azure.ResourceManager.RedisEnterprise/api/Azure.ResourceManager.RedisEnterprise.net8.0.cs

```diff
@@ -959,4 +995,27 @@ protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer
         string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.RedisEnterprise.Models.RedisPersistenceSettings>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
         System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.RedisEnterprise.Models.RedisPersistenceSettings>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
     }
+    public partial class SkuDetails : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.RedisEnterprise.Models.SkuDetails>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.RedisEnterprise.Models.SkuDetails>
```

Comment:
I think we should rename this to `RedisEnterpriseSkuDetails`.
Please do it by adding a key value pair to the `rename-mapping` section in `autorest.md` file and then regenerate the code, such as:
```
rename-mapping:
    SkuDetails: RedisEnterpriseSkuDetails
```